Dockerfile

```diff
@@ -11,7 +11,22 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-FROM --platform=$BUILDPLATFORM golang:1.17.8 as builder-env
+FROM --platform=$BUILDPLATFORM ghcr.io/oracle/oraclelinux:8-slim as builder-env
+
+ENV GOPATH=/root/go
+ENV PATH=$PATH:/usr/local/go/bin/
+ENV PATH=$PATH:${GOPATH}/bin/
+
+RUN mkdir -p ${GOPATH}/src && mkdir -p ${GOPATH}/bin && \
+    microdnf update -y && \
+    microdnf clean all && \
+    rm -rf /var/cache/yum/*
+
+RUN microdnf install git wget tar gzip bzip2
+
+RUN wget https://go.dev/dl/go1.17.5.linux-amd64.tar.gz \
```

Comment:
Do we know whether 1.17.5 is an approved version of Golang?

Reply:
yes its approved . check here - https://github.com/verrazzano/verrazzano-monitoring-operator/blob/master/docker-images/verrazzano-monitoring-operator/Dockerfile

Reply:
Ok good. In that case, please use the same yum repo to install it here (using microdnf install)